eslint-config-base/index.js

```diff
@@ -55,12 +55,94 @@ module.exports = {
       'exports': 'always-multiline',
       'functions': 'only-multiline',
     }],
+    'array-callback-return': [2, { allowImplicit: true }],
+    'complexity': [2, 10],
+    'default-case': [2, { commentPattern: '^no default$' }],
+    'default-case-last': 2,
+    'guard-for-in': 2,
+    'max-classes-per-file': [2, 1], 
+    'no-alert': 2,
+    'no-else-return': [2, { allowElseIf: false }],
+    'no-empty-function': [2, {
+      allow: [
+        'arrowFunctions',
+        'methods'
+      ]
+    }],
+    'no-implicit-coercion': [2, {
+      boolean: false,
+      number: true,
+      string: true,
+      allow: []
+    }],
+    'no-magic-numbers': [2, {
+      ignore: [],
+      ignoreArrayIndexes: true,
+      enforceConst: true,
+      detectObjects: false
+    }],
+    'no-param-reassign': [2, {
+      props: true,
+      ignorePropertyModificationsFor: [
```

Comment:
this react router and express keywords in base especially 😕
I do not know if we need them even in react config

Reply:
These rules are not only for react but they are configured to avoid issues with the react-router library and others. We also use express for our BFFs, so these rules are needed

Reply:
If its not valid for most of projects, we can add this rule as override to our project. Base config should be "base" 😀